cloud/blockstore/libs/diagnostics/profile_log.cpp

```diff
@@ -126,6 +128,11 @@ class TProfileLog final
     void DoFlush();
 };
 
+TProfileLog::~TProfileLog()
+{
+    Flush();
```

Comment:
in blockstore for some reason the implementation of Flush looks like this
```c++
bool TProfileLog::Flush()
{
    if (AtomicGet(ShouldStop)) {
        return false;
    }

    DoFlush();
    return true;
}
```
so it won't do anything since on service stop ShouldStop is set ... you better call DoFlush() instead